### PR TITLE
feat(cms): Add homeCallout object to "home_copy"

### PR DESCRIPTION
**Pull request by dunphyben**

### CODE REVIEW

File: static/admin/config.yml [hunk 1]
```diff
@@ -37,10 +37,26 @@ collections: # A list of collections the CMS should be able to edit
         file: "content/copy/home.yml"
         description: "Copy for the home page"
         fields:
+          - name: homeCallout
```
Comment (BerkeleyTrue) [inline static/admin/config.yml]: You don't need to qualify this as `home` since this is already in the home copy file

File: static/admin/config.yml [hunk 2]
```diff
@@ -37,10 +37,26 @@ collections: # A list of collections the CMS should be able to edit
...
+            label: "Left Callout"
+            widget: object
+            fields:
+
+                - name: title
+                  label: Home Callout Title
```
Comment (BerkeleyTrue) [inline static/admin/config.yml]: Same with these, They will be subheader fields in the editor and should be obvious it is the title of the callout on the home page

File: static/admin/config.yml [hunk 3]
```diff
@@ -37,10 +37,30 @@ collections: # A list of collections the CMS should be able to edit
...
+            widget: object
+            fields:
+
+                - name: title
+                  label: Title
+                  widget: string
+                  tagname: h1
+
+                - name: paragraph
+                  label: Paragraph
```
Comment (BerkeleyTrue) [inline static/admin/config.yml]: Instead of paragraph let's call this copy or description

Comment (dunphyben) [reply static/admin/config.yml]: I think paragraph is more descriptive for the copy editor. It tells them that the copy should be structured in sentence format and limited to 1 paragraph.

Comment (BerkeleyTrue) [reply static/admin/config.yml]: I've never seen that, though. How about `body`?

File: static/admin/config.yml [hunk 4]
```diff
@@ -28,19 +28,39 @@ collections: # A list of collections the CMS should be able to edit
                 label: Title
                 widget: string
 
-              - name: description
-                label: Description
```
Comment (BerkeleyTrue) [inline static/admin/config.yml]: If you are going to change this you are going to have to change it in the components themselves. You should confine the scope of this PR to the copy of the home page only and do this in a separate PR
